benchmark_v2/io_benchmark.py

```diff
@@ -0,0 +1,259 @@
+
+        a = _generate_df(N, dtype, True)
+
+        benchmark.pedantic(_append_files, args=(a, dtype, comp), rounds=pytest.trials)
+
+        nbytes = 0
+        for col in a.values():
+            if dtype in ["int64", "float64", "uint64"]:
+                nbytes = col.size * col.itemsize * pytest.io_files
+            else:
+                nbytes = col.nbytes * col.entry.itemsize * pytest.io_files
+
+        benchmark.extra_info[
+            "description"
+        ] = f"Measures the performance of IO write {dtype} to Parquet file using {comp} compression"
+        benchmark.extra_info["problem_size"] = pytest.prob_size
+        benchmark.extra_info["transfer_rate"] = "{:.4f} GiB/sec".format(
+            (nbytes / benchmark.stats["mean"]) / 2**30
+        )
+
+
+@pytest.mark.benchmark(group="Arkouda_IO_Read_HDF5")
+@pytest.mark.parametrize("dtype", TYPES)
+def bench_ak_read_hdf(benchmark, dtype):
+    if pytest.io_read or (not pytest.io_write and not pytest.io_delete):
+        dataset = "strings_array" if dtype == "str" else "array"
+        a = benchmark.pedantic(
+            ak.read_hdf, args=[pytest.io_path + f"_hdf_{dtype}*", dataset], rounds=pytest.trials
+        )
+
+        nbytes = 0
+        if isinstance(a, ak.pdarray):
+            nbytes += a.size * a.itemsize
+        elif isinstance(a, ak.Strings):
+            nbytes += a.nbytes * a.entry.itemsize
+
+        benchmark.extra_info["description"] = "Measures the performance of IO read from HDF5 files"
+        benchmark.extra_info["problem_size"] = pytest.prob_size
+        benchmark.extra_info["transfer_rate"] = "{:.4f} GiB/sec".format(
+            (nbytes / benchmark.stats["mean"]) / 2**30
+        )
+
+
+@pytest.mark.benchmark(group="Arkouda_IO_Read_Parquet")
+@pytest.mark.parametrize("dtype", TYPES)
+@pytest.mark.parametrize("comp", COMPRESSIONS)
+def bench_ak_read_parquet(benchmark, dtype, comp):
+    if pytest.io_read or (not pytest.io_write and not pytest.io_delete) and comp in pytest.io_compression:
+        a = benchmark.pedantic(
+            ak.read_parquet, args=[pytest.io_path + f"_par_{comp}_{dtype}_*"], rounds=pytest.trials
+        )
+
+        nbytes = 0
+        if isinstance(a, ak.pdarray):
+            nbytes += a.size * a.itemsize
+        elif isinstance(a, ak.Strings):
+            nbytes += a.nbytes * a.entry.itemsize
+
+        benchmark.extra_info["description"] = "Measures the performance of IO read from Parquet files"
+        benchmark.extra_info["problem_size"] = pytest.prob_size
+        benchmark.extra_info["transfer_rate"] = "{:.4f} GiB/sec".format(
+            (nbytes / benchmark.stats["mean"]) / 2**30
+        )
+
+
+@pytest.mark.benchmark(group="Arkouda_IO_Read_Parquet")
+@pytest.mark.parametrize("dtype", TYPES)
+@pytest.mark.parametrize("comp", COMPRESSIONS)
+def bench_ak_read_parquet_multi_append(benchmark, dtype, comp):
+    """
+    Read files written by parquet multicolumn and parquet append modes
+    """

```

Comment:
Can we update the name and the description? I understand the idea here, but we are only the idea is reading a parquet file with multiple columns. It looks like we are only reading only one set of the files (as I would expect). Maybe `bench_ak_read_parquet_multiple_columns`